logger/logger.go

```diff
@@ -231,6 +251,8 @@ func (l *Logger) logWithLevel(logLvl LogLevel, data map[string]interface{}) {
 	}
 	if l.logRouter != nil {
 		data["_kvmeta"] = l.logRouter.Route(data)
+	} else if globalRouter != nil {
+		data["_kvmeta"] = globalRouter.Route(data)
 	}
```

Comment:
I wonder if there will be any confusion that maybe globalRouter is to be combined with the local router